server/controllers/mobile-phone-token-controller.js

```diff
@@ -61,6 +61,60 @@ export const createMobilePhoneToken = (req, res) => {
   }
 };
 
+export const mobilePhoneTokenStatus = (req, res) => {
+  const reqOrg = req.params.organization;
+  const validSlug = config.some((org) => {
+    if (org.slug === reqOrg) {
+      // merge default config and custom config
+      const conf = merge(defaultConfig, org);
+      const {host} = conf;
+      const url = reverse("mobile_phone_token_status", getSlug(conf));
+      const timeout = conf.timeout * 1000;
+      // make AJAX request
+      axios({
+        method: "get",
+        headers: {
+          "content-type": "application/x-www-form-urlencoded",
+          Authorization: req.headers.authorization,
+          "accept-language": req.headers["accept-language"],
+        },
+        url: `${host}${url}/`,
+        timeout,
+      })
+        .then((response) => {
+          res
+            .status(response.status)
+            .type("application/json")
+            .send(response.data);
+        })
+        .catch((error) => {
+          logResponseError(error);
+          // forward error
+          try {
+            res
+              .status(error.response.status)
+              .type("application/json")
+              .send(error.response.data);
+          } catch (err) {
+            res.status(500).type("application/json").send({
+              response_code: "INTERNAL_SERVER_ERROR",
+            });
+          }
+        });
+    }
+    return org.slug === reqOrg;
+  });
+  // Return 401 for invalid organization slug or org not listed in config.
+  // This is different from other implementation because the frontend
+  // expects 404 only when the phone token status API endpoint is
+  // not implemented in OpenWISP RADIUS.
+  if (!validSlug) {
+    res.status(401).type("application/json").send({
+      response_code: "UNAUTHORIZED",
+    });
```

Comment:
Instead of using 404, I used 401 which makes it inconsistent with other views. 

When doing manual testing, if you visit http://localhost:8080/default/mobile-phone-verification for a random organization, it would show 404 page before this code is even executed.

Reply:
I have updated this code again.